.github/workflows/coverage.yml

```diff
@@ -96,7 +96,11 @@ jobs:
           ctest -VV;
           (cd ..; lcov --capture  --directory . --no-external --output-file build/coverage-run.info)
           lcov --add-tracefile coverage-base.info --add-tracefile coverage-run.info --output-file coverage-combined.info
-
+          
+          # Print out caliper data
+          echo "cali.out:"
+          cat `find ../* -name cali.out`
```

Comment:
`nitpick`: to my eyes, cleaner solution would be to not have `CALI_CONFIG=runtime-report` added to `TEST_ENV`. (i.e. have another env var where profiling is not added and then use it for `nrntest`)

Reply:
Great idea, will try that.